core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/SequencerReference.java

```diff
@@ -31,14 +32,30 @@
 import org.codehaus.jackson.map.annotate.JsonSerialize;
 
 /**
- * A SequencerReference is a {@link HardwareReference} specifically designated with a {@link Platform}
+ * A SequencerReference is a {@link SequencerReference} specifically designated with a {@link Platform}
```

Comment:
This implies the possibility of a SequencerReference that is not a SequencerReference :grinning: